package_build_list_host_darwin.json

```diff
@@ -63,8 +63,8 @@
         "Lua-5.4.4-rev1-ios": "package-system/Lua/temp/Lua-ios",
         "freetype-2.11.1-rev1-mac": "package-system/freetype/temp/freetype-mac",
         "freetype-2.11.1-rev1-ios": "package-system/freetype/temp/freetype-ios",
-        "googlebenchmark-1.5.0-rev2-ios": "package-system/googlebenchmark-ios",
-        "googlebenchmark-1.5.0-rev2-mac": "package-system/googlebenchmark-mac",
+        "googlebenchmark-1.7.0-rev1-ios": "package-system/googlebenchmark-ios",
+        "googlebenchmark-1.7.0-rev1-mac": "package-system/googlebenchmark-mac",
```

Comment:
Since you're in here anyway, maybe consider updating this to 
```
package-system/googlebenchmark/temp/googlebenchmark-ios
package-system/googlebenchmark/temp/googlebenchmark-mac
```
up here and in the build script above, similar to the others.   Its the 'package root' param, can just be set to ../../package-system/googlebenchmark/temp

Then the temp folder will match the already existing git exclusion for temp

Reply:
Actually there was an update to the base script so the above section can omit the --package-root parameter, it will default to the first argument. (see suggestion above)

Reply:
```suggestion
        "googlebenchmark-1.7.0-rev1-ios": "package-system/googlebenchmark/temp/googlebenchmark-ios",
        "googlebenchmark-1.7.0-rev1-mac": "package-system/googlebenchmark/temp/googlebenchmark-mac",
```